cypress/e2e/bookDetailView.cy.ts

```diff
@@ -0,0 +1,50 @@
+import { createBook } from '../factories/bookFactory'
+
+const book = createBook()
+describe('Test book detail page', () => {
+  beforeEach(() => {
+    cy.intercept('GET', '/books/isbn/*', book).as('getBook')
```

Comment:
[SUG] Following the [best practices](https://docs.cypress.io/guides/references/best-practices) from cypress, we should add a custom command to avoid repeatedly using  `cy.get('[data-test-id="your-test-id"]')`

This should be the command to add in `support/commands.ts`:
```.ts
Cypress.Commands.add('getByTestId', (selector, ...options) => {
  return cy.get(`[data-test-id=${selector}]`, ...options)
})
```

Then you will have to create a file `support/index.d.ts` and add the new command in the interface so the code editor doesn't complain. Filling it with this should be fine:
```.ts
declare namespace Cypress {
  interface Chainable {
    /**
     * @param selector The data-test-id attribute value
     * @example cy.getByTestId('book-detail')
     */
    getByTestId(selector: string): Chainable<JQuery<HTMLElement>>
  }
}

```


With this setup, you can now write:
```.ts
cy.getByTestId('book-detail').should('be.visible')
```
And it looks more readeable

Reply:
Looks like a good idea, I'll apply it